hs628_tools/Preprocess_module.py

```diff
@@ -0,0 +1,144 @@
+
+# coding: utf-8
+
+# In[233]:
+
+
+get_ipython().magic(u'matplotlib inline')
+
+
+# In[234]:
+
+
+
+
+import matplotlib
+import numpy as np
+import pandas as pd
+import matplotlib
+import matplotlib.pyplot as plt
+#pd.test()
+
+
+# In[235]:
+
+
+df1 = pd.read_csv("C:/Users/Nitie/Desktop/health_informatics/MSHI_CLASSES/summer_2018/650/project data/parkinsons_train_set.csv")
+
+df1.head(10)
+
+
+# In[236]:
+
+
+
+
+plt.hist(df1["HNR"], normed=True, bins=20)
+plt.ylabel('FREQ');
+plt.grid()
+plt.rc('grid', linestyle="dashed", color='grey')
+
+
+
+# In[237]:
+
+
+cat=df1["status"].unique()
+cat
+
+
+# In[238]:
+
+
+df1[["HNR","status"]]
+
+
+# In[239]:
+
+
+df1["HNR_status"] =list(zip(df1.HNR, df1.status))
+df1["HNR_status"].head(10)
+
+
+# Build a tuple of the column and class label
+
+# In[240]:
+
+
+
+def coltuple(df,col,typpe):
```

Comment:
What is the parameter "typpe" referring to ?

Reply:
The third parameter "typpe" is the column name of the categorical variable e.g class label (mortality).